JavaScript Grocery List Project/js/app.js

```diff
@@ -1,22 +1,28 @@
+//Event handler for enable list
+document.addEventListener("DOMContentLoaded", initList);
 
-(function(){
+function initList(){
 
+    
     let listHolder = [];    
     
+    //get data from local storage
+    function getOldData(){    
+         let oldList = localStorage.getItem('listHolder');
+            //console.log(oldList);
+         if(oldList){
+                listHolder = JSON.parse(oldList);
+                addItem(listHolder);
+         }
+     }
+    getOldData();
+    
     //Function for save to local
     function saveToLocal(data){
         localStorage.setItem('listHolder', JSON.stringify(data));

```

Comment:
can you give more explanation on using of Localstorage? do you have any other option for similar functionalities

Reply:
Hi Sanjay Sir,
Localstorage object save data with no expiration date. even if you close browser window. We will be find out the local storage value on the next day, week or after months and year. 
I am used 3 methods of Localstorge
1. localStorage.setItem(); // For set value
2: localStorage.getItem(); // Get value from storage
3. localStorage.removeItem(); //For remove specific data we saved to localStorage.

Yes we have another option is sessionStorage for save data browser storage, I am not used it because of we will lose our data when we close tab.